### ISSUE TITLE
Build failed on android 

### ISSUE THREAD

**Issue by saidubundukamara** [thevikke/read_pdf_text#10]
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':read_pdf_text'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 1m 1s
Error: Gradle task assembleDebug failed with exit code 1

**Comment by thevikke** [comment]
@saidubundukamara I was able to reproduce the issue, but coming up with a proper fix might take a couple of days. Here is a quick solution. Downgrade the APG version of your Flutter app by lowering the com.android.application version in android\settings.gradle.

```
// settings.gradle

<<other code>>

plugins {
    id "dev.flutter.flutter-plugin-loader" version "1.0.0"
    id "com.android.application" version "7.3.0" apply false  <- set this to 7.3.0
    id "org.jetbrains.kotlin.android" version "1.8.22" apply false
}

include ":app"
```

**Comment by mohsinnaqvi606** [comment]
@thevikke 
Any update or solution for this?
I can not downgrade the APG version, this will cause issues with other plugins.

**Comment by thevikke** [comment]
Yes I have a solution in the works but I am on a work trip so it will probably be a week before I have time to finish it. Sorry!

**Comment by thevikke** [comment]
@mohsinnaqvi606 Perhaps you can try downgrading Flutter and see if that helps, as this is caused by upgrading Flutter.

**Comment by mohsinnaqvi606** [comment]
@thevikke No, Downgrading is not possible. I will wait for your next update.
for now skipping this feature in the App

**Comment by thevikke** [comment]
I have released a new version, 0.2.3, which includes the AGP fix and several other upgrades. When I tested it with a fresh Flutter app using the command `flutter create`, I encountered an "androidJdkImage" error, similar to the one described in this [issue](https://github.com/fluttercommunity/plus_plugins/issues/3303). If you run into similar problems, please follow the instructions provided in that issue.
